### DISCUSSION TITLE
The listing is not editable, must be active or expired but is removed.

### DISCUSSION THREAD

**Discussion by FatihGokmen** [etsy/open-api#969] [Q&A]
Hello,

We want to actively re-sell the product in the ad. We couldn't get it from api's. We get the following warning as an error.

'List cannot be edited, must be active or expired, but has been removed.'

How should we proceed to reactivate the product?

respects.

**Comment by manasseh-fikadu** [comment]
Hey @FatihGokmen have got the solution to this?

**Comment by showtroylove** [comment]
I need more details to assist you properly. Have you tried recreating the listing directly using Etsy’s listing form? If so, were you able to publish it successfully?

**Comment by amitkumar-dev92** [comment]
Hello @FatihGokmen ,
It seems that listing_id is already deleted before you are trying to activate or edit that. 
if you are using any 3rd party API to manage listing, it's worth to connect with them. Otherwise for few listing_ids you can manually verify listing_status from Etsy.com shop manage section.
if you could share listing_id, it would be easier to guide better.
thanks
